spec/std/isa/isa/globals.isa

```diff
@@ -874,9 +874,9 @@ function xlen {
       return 32;
     } else {
       if (mode() == PrivilegeMode::M) {
-        if (CSR[misa].MXL == $bits(XRegWidth::XLEN32)) {
+        if ((!MISA_CSR_IMPLEMENTED) || (CSR[misa].MXL == $bits(XRegWidth::XLEN32))) {
           return 32;
-        } else if (CSR[misa].MXL == $bits(XRegWidth::XLEN64)) {
+        } else if ((!MISA_CSR_IMPLEMENTED) || (CSR[misa].MXL == $bits(XRegWidth::XLEN64))) {
           return 64;
         }
```

Comment:
I'm not quite sure here, but pretty sure this needs to change. This always returns 32 if `!MISA_CSR_IMPLEMENTED`. @dhower-qc ?